where-to-travel/src/main/webapp/login.js

```diff
@@ -0,0 +1,21 @@
+// Get elements for authentication
+const emailLogin = document.getElementById('emailLogin');
+const passwordLogin = document.getElementById('passwordLogin');
+const btnLogin = document.getElementById('login');
+
+// Add login event
+btnLogin.addEventListener('click', e => {
+  const email = emailLogin.value;
+  const pass = passwordLogin.value;
+  const auth = firebase.auth();
+  
+  // Login 
+  const promise = auth.signInWithEmailAndPassword(email, pass);
+  promise.then(e => {
+    alert("You have sucessfully logged in!");
+  });
+  promise.catch(e => {
+    console.log(e.message);
+    alert(e.message);
```

Comment:
What is e? Will alerting the user with e help them to complete a successful login?